### DISCUSSION TITLE
Focus selected data

### DISCUSSION THREAD

**Discussion by melarenina** [plouc/nivo#1648] [Q&A] [answered]
Hi,

Is it possible to replicate a behavior like this, where the chart focus only the selected data?

![2021-06-30_13h25_33](https://user-images.githubusercontent.com/38476041/123997718-8a5a8d00-d985-11eb-96c6-c8486099f179.gif)

Thank you.

**Comment by wyze** [comment, marked as answer]
Something like this? https://codesandbox.io/s/sparkling-dream-f2pyo?file=/src/SelectableBarLayer.js

**Reply by melarenina** [reply]
Yes! Thank you very much! :)

**Reply by melarenina** [reply]
@wyze Do you have any example for pie chart or line? Thanks!

**Reply by wyze** [reply]
Here is one I put together for the Pie chart for another discussion. https://codesandbox.io/s/vibrant-wescoff-txxtm?file=/src/App.tsx
